### DISCUSSION TITLE
Datasets: Diff of output and expected output

### DISCUSSION THREAD

**Discussion by marcklingen** [langfuse/langfuse#1008] [Ideas]
When running experiments on datasets, I compare a lot of texts. It would be useful to have a diff of expected outputs and expected outputs.

- change of run table to include: input, output, expected output
- toggle to enable diff on output and expected output

Tracking this here to understand who else might be interested in this.

**Comment by c0pper** [comment]
Same, in langsmith I can open a dataset run and see input, output and expected output. Here when I open a dataset run I only see random ids, not really useful. For this reason, when i run an experiment on a dataset i log it to a dedicated session where at least i can see the input and the model output in one view

**Comment by andreieuganox** [comment]
Is there any real reason not to have it? Without this, the whole thing relies only on Scores you calculate, but you can not see why that score is low at a glance.

**Comment by andreieuganox** [comment]
@marcklingen can you re-open original Issue, PR is on the way: https://github.com/langfuse/langfuse/pull/4279

**Reply by marcklingen** [reply]
thanks for the PR, feel free to reference this discussion instead of the issue

**Reply by marcklingen** [reply]
really excited about this, will get to this post-launch-week
